slider/position.py

```diff
@@ -37,3 +38,7 @@ class Point(namedtuple('Point', 'x y offset')):
     The visible region of the osu! standard playfield is [0, 512] by [0, 384].
     Positions may fall outside of this range for slider curve control points.
     """
+
+
+def distance(start, end):
```

Comment:
This should have a docstring like the other functions and methods in slider. This helps users and developers know how to use the function. In my opinion, this is useful even for small functions like this.